src/App.css

```diff
@@ -17,14 +17,26 @@
 }
 
 .App-footer {
-	padding: 10px;
-	margin: 10px;
-	background-color: #cce6ff;
+  position:fixed;
```

Comment:
fixed position cause another problem, it will hide the content of app container.
try to add some text in container, and that will hide the buttons